components/Layout.jsx

```diff
@@ -1,12 +1,13 @@
 import Logo from './Logo';
 import SkipToContent from './SkipToContent';
 
-export default function Layout({ children }) {
+export default function Layout({ children, skippedComponent }) {
   return (
     <div className="obj-layout">
       <header>
         <SkipToContent />
         <Logo />
+        {skippedComponent}
```

Comment:
Yeah, curious to know about this change. Given that the Filters aren't on every page, I probably wouldn't consider them part of the `<header>` since that piece is usually constant across the site, both for user experience and for accessibility.

Reply:
The reasoning behind this change was so that the SkipToContent link will skip the filters and go straight to the content. Why I put it in the `<header>` because I was getting an axe dev tool warning about content being contained in landmarks. Now that you said something I'm thinking it would probably be more appropriate to include in a `<section>` and include some conditional logic so that the section only appears on the overview page

Reply:
Oh yeah, good call skipping over the filters! I think that making this a prop might be an unnecessary layer of abstraction. Based on the designs it seems like the only component that will ever need to be skipped (other than the header) is the filters, is that right? So I think we can put the Filters component directly under the `<header>` wrapped in a `<section>` like you said so that the code is more readable for someone new to the code. Let me know what you think!

Reply:
Going to resolve this! Per our zoom convo, we want to stay away from prop drilling and this solution doesn't require extra conditionals to turn filters on/off per page ✨